change/@fluentui-react-charting-594b823b-95eb-4e54-968b-d87bc609a723.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fetching theme attributes from tokens",
+  "packageName": "@fluentui/react-charting",
+  "email": "t-agarwalp@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
Add description on the PR page about the feature, any known issues and pending tasks.